src/ui/app.rs

```diff
@@ -72,6 +72,15 @@ impl WalksnailOsdTool {
         let srt_options = config.srt_options;
         let osd_options = config.osd_options;
 
+        // Load last used font file
+        let mut font_file = None;
+        let font_path = PathBuf::from(config.font_path);
+        if font_path.exists() && font_path.is_file() {
+            if let Ok(f) = font::FontFile::open(font_path) {
+                font_file = Some(f);
+            }
+        }
```

Comment:
The path not existing or being a directory instead of a file is already handled by `font::FontFile::open()` so this can be simplified to:
```suggestion
        let font_path = PathBuf::from(config.font_path);
        let font_file = font::FontFile::open(font_path).ok();
```